java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -703,7 +703,11 @@ void testSequenceOtherTypes() {
 
   @Test
   void testFromScalarZeroRows() {
-    for (DType type : DType.values()) {
+    for (DType.DTypeEnum type : DType.DTypeEnum.values()) {
+      // Decimal type not supported yet. Update this once it is supported.
+      if (type == DType.DTypeEnum.DECIMAL32 || type == DType.DTypeEnum.DECIMAL64) {
+        continue;
```

Comment:
Is this going to be a follow on issue?

Reply:
I was hoping this could be covered in this https://github.com/rapidsai/cudf/issues/6515 as well. But if you think these things should not be covered in umbrella bug, I will create another issue. Please let me know.

Reply:
OK